perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -105,6 +105,30 @@
 	 * @throw MemberNotExistsException When Member with <code>id</code> doesn't exist.
 	 * @throw ResourceNotExistsException When Resource with <code>id</code> doesn't exist.
 	 */
+	/*#
+	 * Get all attributes by the list of attrNames if they are in one of these namespaces:
+	 * - member
+	 * - group
+	 * - member-group
+	 * - resource
+	 * - member-resource
+	 * - group-resource
+	 * - user (get from member object)
+	 * - facility (get from resource object)
+	 * - user-facility
+	 *
+	 * @param member int Member <code>id</code>
+	 * @param group int Group <code>id</code>
+	 * @param resource in Resource <code>id</code>
+	 * @param attrNames List<String> Attribute names
+	 * @return List<Attribute> All attributes from supported namespaces.
+	 * @throw MemberNotExistsException When Member with <code>id</code> doesn't exist.
+	 * @throw ResourceNotExistsException When Resource with <code>id</code> doesn't exist.
+	 * @throw GroupNotExistsException When Group with <code>id</code> doesn't exist.
+	 * @throw MemberResourceMismatchException When Member is not from the same Vo as Resource.
+	 * @throw GroupResourceMismatchException When Group is not from the same Vo as Resource.
+	 * @throws InternalErrorException When any internal error occurs.
```

Comment:
This exception is added by default by our docs generator.

Reply:
Removed.